adb/contrib/adb_commands_safe.py

```diff
@@ -433,8 +433,11 @@ def ShellRaw(self, cmd):
     # The adb protocol doesn't return the exit code, so embed it inside the
     # command.
     assert self.IsShellOk(cmd), 'Command is too long: %r' % cmd
-    out = self._adb_cmd.Shell(cmd + self._SHELL_SUFFIX).decode(
-        'utf-8', 'replace')
+    timeout_ms = max(
+        timeout_ms or self._default_timeout_ms, self._default_timeout_ms)
```

Comment:
This might actually not be necessary, but I'm not totally clear on all of the ways that the USB handle can be created.  If it has the same default timeout as the AdbCommandsSafe object, then this logic is unnecessary.